packages/lesswrong/components/forumEvents/ForumEventPoll.tsx

```diff
@@ -16,26 +16,26 @@ import { Link } from "@/lib/reactRouterWrapper";
 import { postGetPageUrl } from "@/lib/collections/posts/helpers";
 import { commentGetPageUrlFromIds } from "@/lib/collections/comments/helpers";
 import { parseDocumentFromString, ServerSafeNode } from "@/lib/domParser";
+import { PartialDeep } from "type-fest";
+import { stripFootnotes } from "@/lib/collections/forumEvents/helpers";
+import { useMessages } from "../common/withMessages";
+import { useSingle } from "@/lib/crud/withSingle";
 
-export const POLL_MAX_WIDTH = 800;
 const SLIDER_MAX_WIDTH = 1120;
 const RESULT_ICON_MAX_HEIGHT = 27;
-const USER_IMAGE_SIZE = 34;
+const USER_IMAGE_SIZE = 30;
 const DEFAULT_STACK_IMAGES = 20;
-const NUM_TICKS = 29;
+const NUM_TICKS = 19;
```

Comment:
My solution to making it work on mobile is just to adjust these to values that look ok on both desktop and mobile